src/constants/theme.js

```diff
@@ -1 +1,37 @@
 export const accent = '#f90000';
+
+const colorPallet = {
```

Comment:
So this is great. I like it. Even though I'm not asking to making up collor pallets, please take visual key document as reference and use the palletes, (hues of different colors) and declare them like [these people do](https://github.com/keegn/gatsby-starter-saas-marketing/blob/master/src/styles/theme.js#L25)